plugin-build/src/main/kotlin/io/sentry/android/gradle/instrumentation/AbstractSpanAddingMethodVisitor.kt

```diff
@@ -41,23 +41,45 @@ abstract class AbstractSpanAddingMethodVisitor(
     }
 
     /*
-    ISpan span = Sentry.getSpan()
+    IHub hub = HubAdapter.getInstance();
+    ISpan span = hub.getTransaction() ||  hub.getSpan();
     ISpan child = null;
     if (span != null) {
       child = span.startChild("db", <description>);
     }
     */
     protected fun MethodVisitor.visitStartSpan(
+        attachToRoot: Boolean,
```

Comment:
Not sure if we need this flag. But in theory we could always attach the rooms spans to the root and the sqlite spans to the last active span

Reply:
yeah, I think I'd just do that without the flag